### PR TITLE
add lsp-disable and lsp-disable-window

### PR DESCRIPTION
**Pull request by andreyorst**
As suggested in https://github.com/ul/kak-lsp/issues/228 I've added two commands `lsp-disable` and `lsp-disable-window`. However after executing those hovering still works for some reason. Did I missed something?

### CODE REVIEW

**Comment by andreyorst** [comment]
ah. found that. Interestingly enough that `lsp-auto-hover` hooks are always `global` scoped. Can't this cause problems with `window` scoped lsp?

File: rc/lsp.kak [hunk 1]
```diff
@@ -961,8 +975,8 @@ define-command lsp-enable-window -docstring "Default integration with kak-lsp in
     lsp-inline-diagnostics-enable window
     lsp-diagnostic-lines-enable window
 
-    map window goto d '<esc>: lsp-definition<ret>' -docstring 'definition'
-    map window goto r '<esc>: lsp-references<ret>' -docstring 'references'
+    map window goto d '<esc>: lsp-definition<ret>'
```
Comment (ul) [inline rc/lsp.kak]: Any specific reason to remove docstring here and below?

**Comment by ul** [comment]
> Interestingly enough that lsp-auto-hover hooks are always global scoped. Can't this cause problems with window scoped lsp?

Yes, I think I overlooked it when introduced `lsp-enable-window`. Most likely we want `auto` hooks to be scope-parametrized as well.

**Comment by ul** [comment]
Thanks for adding this command! Let's merge it (after restoring docstrings) and then have `auto` commands fixed as well.

File: rc/lsp.kak (hunk 1 above)
Comment (andreyorst) [reply rc/lsp.kak]: Oi!

**Comment by andreyorst** [comment]
I've put docstrings back. It seems that I've copied map and unmap commands around and unmap doesn't accept docstring, and I've removed those docstrings by mistake.

**Pull request merged**